### ISSUE TITLE
maintenance

### ISSUE THREAD

**Issue by mikeatm** [nitely/python-react-v8#8]
Is this still being maintained?

**Comment by nitely** [comment]
Yes, it's. There's no activity because it's "done" and no one has reported any issue. I've been using it in a project for years now.
